Libraries/Image/ImageProps.js

```diff
@@ -98,6 +98,28 @@ export type ImageProps = {|
    */
   capInsets?: ?EdgeInsetsProp,
 
+  /**
+   * Adds the CORS related header to the request.
+   * Similar to crossorigin from HTML.
+   *
+   * See https://reactnative.dev/docs/image#crossorigin
+   */
+  crossOrigin?: ?('anonymous' | 'use-credentials'),
+
+  /**
+   * Height of the image component.
+   *
+   * See https://reactnative.dev/docs/image#height
+   */
+  height?: ?number,
+
+  /**
+   * Width of the image component.
+   *
+   * See https://reactnative.dev/docs/image#width
+   */
+  width?: ?number,
```

Comment:
Hi @dhruvtailor7. We can't merge this PR internally because our `width` prop is of type `string | number`. Also, I think that if we use `width?` to define an optional property, there is no need to use an optional type for the value (`?number`).

Could you update the PR so that I can try to import it again? Thank you!

Reply:
@cipolleschi does the same apply to height too?

Reply:
@cipolleschi I have updated the type to match the type from the style prop.

Reply:
The internal failure was because a component was defining `width` for custom purposes and didn't strip it from the props it forwarded to the RN components. We probably shouldn't match it's type because the DOM type is `number`: https://github.com/facebook/flow/blob/v0.111.3/lib/dom.js#L3025

Reply:
@necolas So, I should just revert the type to `number` and do [these](https://github.com/facebook/react-native/pull/34481/files/acbe420b9be2c20c249d287ea674dc52efbbe52a#r963292725) changes, right?

Reply:
Yes the type should be `number`